code/src/cljs/sixsq/nuvla/ui/deployments/events.cljs

```diff
@@ -62,15 +62,17 @@
   (fn [{{:keys [::spec/additional-filter
                 ::spec/state-selector
                 ::spec/filter-external
-                ::spec/ordering] :as db} :db} [_ {:keys [filter-external-arg pagination-db-path]}]]
+                ::spec/ordering] :as db} :db} [_ {:keys [filter-external-arg pagination-db-path external-filter-only?]}]]
```

Comment:
Did you checked that the problem is only happening on apps pages?
I think that get-deployments is used in different pages like edge  and deployment set.

Reply:
I started doing this with the full knowledge that it is used elsewhere and the clear goal to check all usages. And then I just forgot about.

Now after checking it, I noticed another usage deep down in the VerticalDeploymentsTable, where we pass down the fetch-event to the sort-config:

```
:sort-config {:db-path     ::spec/ordering
                       :fetch-event ::events/get-deployments}
```

So when you click a sort column it applies the saved filters again, hmpf.

Fixing it now